Mathlib/LinearAlgebra/Matrix/ToLin.lean

```diff
@@ -1008,3 +1008,39 @@ def algEquivMatrix [Fintype n] (h : Basis n R M) : Module.End R M ≃ₐ[R] Matr
 #align alg_equiv_matrix algEquivMatrix
 
 end
+
+namespace Basis
+
+variable {R M ι : Type*} [CommRing R] [AddCommGroup M] [Module R M] [Fintype ι] [DecidableEq ι]
+
+/-- The standard basis of the endomorphism algebra of a module
+induced by a basis of the module.
+
+If `M` is a module with basis `b` indexed by a finite type `ι`,
+then `Basis.end b` is the basis of `Module.End R M` indexed by `ι × ι`
+where `(i, j)` indexes the linear map that sends `b j` to `b i`
+and sends all other basis vectors to `0`.  -/
+@[simps! repr_apply]
+noncomputable
+def _root_.Basis.end (b : Basis ι R M) : Basis (ι × ι) R (Module.End R M) :=
```

Comment:
I think this makes it slightly clearer what the issue is:
```suggestion
def «end» (b : Basis ι R M) : Basis (ι × ι) R (Module.End R M) :=
```
though personally I'd advocate for just calling this `End`.

Reply:
True, I considered it, but otoh didn't really like the french quotes. I don't think it is well-known to non-experts what they do. The current name follows a more widely used pattern.
Do you have a strong preference?

Reply:
No strong preference, let's leave this alone then!